app/services/preview_service.rb

```diff
@@ -44,15 +44,21 @@ def upload_assets(edition)
     edition.image_revisions.each do |image_revision|
       image_revision.ensure_assets
 
-      image_revision.assets.each { |asset| upload_image(edition, asset) }
+      image_revision.assets.each { |asset| upload_asset(edition, asset) }
+    end
+
+    edition.file_attachment_revisions.each do |file_attachment_revision|
+      file_attachment_revision.ensure_assets
```

Comment:
Just wondering why we are calling `ensure_assets` from here and from the file_attachment_upload_service?

Reply:
Yeah I don't think that's needed. We do the same thing with images so I'd want to change them both at the same time (if at all).

Reply:
I've removed them